src/styles/intent.styl

```diff
@@ -0,0 +1,52 @@
+@require 'settings/palette.styl'
+.app-wrapper
+    width 100%
+.app-content
+    height 100vh
+
+.intent-layout
+    height 100%
+    display flex
+    align-items stretch
+    flex-direction column
+
+
+.intent-footer
+    height 4.5rem
+    background-color paleGrey
+    box-shadow inset 0 -1px 0 0 silver
+    display flex
+    align-items center
+    padding .5rem 1rem
+
+    .intent-footer-label
+        flex 1
+
+.intent-main
+    flex 1
+    overflow-y scroll
+    width 100%
```

Comment:
Are the width here and in `intent-header` really useful? Since their parent has `align-items stretch` I think they'll automatically have 100% width.

Reply:
Not sure, I cargo-cult what I found here https://jsfiddle.net/srbakfzw/
And I believe if I removed it, it breaks.

I try to generalize the concept in a codesandbox app you can find here https://codesandbox.io/s/lx4z0yvz9m

Reply:
[Here's a fork of your sandbox](https://codesandbox.io/s/m5zln0rzwy) without the `width 100%` which seems to work the same across all major browsers :)

Reply:
@y-lohse they have automatic height since the flex-direction is column. width is not impacted by this.

Reply:
Erm, no, I don't think so?

Since `flex-direction` is set to `column`, the main axis is vertical. So `justify-content` affects the *main axis*, ie. the `height`, and `align-items` affects the cross-axis, so the `width`.
So setting `align-items stretch` on a flexbox with `flex-direction column` *does* affect the children's width.